src/screens/OtherProfile/index.js

```diff
@@ -679,7 +605,7 @@ const OtherProfile = () => {
           onSelect={onBlocking}
           refBlockUser={blockUserRef}
           username={username}
-          isBlocker={blockStatus.blocker}
+          isBlocker={isBlocking}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `blockStatus.blocker` to `isBlocking` is more readable and understandable.


<!-- This is an auto-generated comment by OSS CodeRabbit -->